### PR TITLE
Implemented: logic to make the job configuration component data dynamic(#23ty1de)

### PR DESCRIPTION
**Pull request by ymaheshwari1**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/hotwax/job-manager#contribution-guideline)

### CODE REVIEW

File: src/views/Orders.vue [hunk 1]
```diff
@@ -225,34 +247,52 @@ export default defineComponent({
...
         this.store.dispatch('job/scheduleService', payload)
       } else if (job?.status === 'SERVICE_PENDING') {
-        payload['tempExprId'] = 'HOURLY'
+        payload['tempExprId'] = 'DAILY'
         payload['jobId'] = job.id
 
         this.store.dispatch('job/updateJob', payload)
       }
+    },
+    openJobConfiguration(enumId: string, title: string, status: string) {
```
Comment (adityasharma7) [inline src/views/Orders.vue:280]:
```suggestion
    viewJobConfiguration(enumId: string, title: string, status: string) {
```

File: src/views/Orders.vue [hunk 2]
```diff
@@ -225,34 +247,52 @@ export default defineComponent({
...
+          'runAsUser': 'system', // default system, but empty in run now
+          'recurrenceTimeZone': ''
+        }
         payload['shopifyConfigId'] = this.getShopifyConfigId
-        payload['productStoreId'] = this.getCurrentEComStore.productStoreId
 
         this.store.dispatch('job/scheduleService', payload)
       } else if (job?.status === 'SERVICE_PENDING') {
-        payload['tempExprId'] = 'HOURLY'
+        payload['tempExprId'] = 'DAILY'
```
Comment (adityasharma7) [inline src/views/Orders.vue:274]: Why this is hard coded?

File: src/views/Product.vue [hunk 3]
```diff
@@ -64,25 +72,38 @@ export default defineComponent({
...
+  computed: {
+    ...mapGetters({
+      getJobStatus: 'job/getJobStatus'
+    }),
   },
   data() {
     return {
       jobEnums: JSON.parse(process.env?.VUE_APP_PRD_JOB_ENUMS as string) as any,
+      currentJobEnum: 'IMP_PRDTS',
+      title: 'Import products'
```
Comment (adityasharma7) [inline src/views/Product.vue:86]: Title should be internationalized

File: src/views/Product.vue [hunk 4]
```diff
@@ -64,25 +72,38 @@ export default defineComponent({
...
       "inputFields":{
-        "serviceName": Object.values(this.jobEnums),
-        "serviceName_op": "in"
+        "systemJobEnumId": Object.values(this.jobEnums),
+        "systemJobEnumId_op": "in"
       }
     });
   },
+  methods: {
+    openJobConfiguration(enumId: string, title: string) {
```
Comment (adityasharma7) [inline src/views/Product.vue:98]:
```suggestion
    viewJobConfiguration(enumId: string, title: string) {
```

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/Orders.vue (hunk 2 above)
Comment (ymaheshwari1) [reply src/views/Orders.vue:274]: Sir, when we are scheduling or updating the jobs where we have toggle or checkbox, then for those jobs we are initializing a hard-coded value for temp expr.

File: src/views/Product.vue (hunk 3 above)
Comment (ymaheshwari1) [reply src/views/Product.vue:86]: Sir, it's internationalized on the JobDetail page, so not internationalizing it here.

File: src/components/JobDetail.vue [hunk 5]
```diff
@@ -1,48 +1,61 @@
...
+      <h1>{{ $t(title) }}</h1>
+      <ion-badge slot="end" color="dark" v-if="job?.runTime">{{ $t("running") }} {{ timeTillJob(job.runTime) }}</ion-badge>
     </ion-item>
 
     <ion-list>
       <ion-item>
         <ion-icon slot="start" :icon="calendarClearOutline" />
         <ion-label>{{ $t("Last run") }}</ion-label>
-        <ion-label slot="end">2:00 PM EST</ion-label>
+        <ion-label slot="end">{{ job?.lastUpdatedStamp ? getTime(job.lastUpdatedStamp) : 'No previous occurrence' }}</ion-label>
```
[the remark this answers is not in the document]
Comment (ymaheshwari1) [reply src/components/JobDetail.vue:12]: Done Sir

File: src/components/JobDetail.vue [hunk 6]
```diff
@@ -1,48 +1,61 @@
...
       </ion-item>
 
       <ion-item>
         <ion-icon slot="start" :icon="timeOutline" />
         <ion-label>{{ $t("Run time") }}</ion-label>
-        <ion-label slot="end">3:00 PM EST</ion-label>
+        <ion-label id="open-run-time-modal" slot="end">{{ job?.runTime ? getTime(job.runTime) : 'Select run time' }}</ion-label>
```
[the remark this answers is not in the document]
Comment (ymaheshwari1) [reply src/components/JobDetail.vue:18]: Done Sir

File: src/components/JobDetail.vue [hunk 7]
```diff
@@ -1,48 +1,61 @@
...
+        <!-- <ion-button id="open-run-time-modal" size="small" fill="outline" color="medium" v-show="!job?.runTime">{{ $t("Select run time") }}</ion-button> -->
```
[the remark this answers is not in the document]
Comment (ymaheshwari1) [reply src/components/JobDetail.vue:19]: Added comment for the same.

File: src/views/Product.vue (hunk 4 above)
Comment (ymaheshwari1) [reply src/views/Product.vue]: Sir, Updated method name

File: src/views/Orders.vue (hunk 1 above)
Comment (ymaheshwari1) [reply src/views/Orders.vue]: Sir, updated method name.

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**